src/lib.rs

```diff
@@ -373,7 +388,7 @@ mod tests {
 
 /// Tests that rely on an external network connection.
 /// Sort of integration tests.
-#[cfg(all(test, network_tests))]
+#[cfg(test)]
```

Comment:
For local testing, this is fine. But I won't let regular unit tests depend on the network (flakyness).

There is a dedicated CI step for that:
https://github.com/phip1611/ttfb/blob/main/.github/workflows/rust.yml#L56